plugin/resurrect/fuzzy_loader.lua

```diff
@@ -15,35 +27,391 @@ function pub.get_default_fuzzy_load_opts()
+		local temp_vbs = os.tmpname() .. ".vbs"
+		local temp_out = os.tmpname() .. ".txt"
+
+		local vbs_script = string.format(
+			[[
+                Set fso = CreateObject("Scripting.FileSystemObject")
+                Set outFile = fso.CreateTextFile("%s", True)
+                
+                Sub ProcessFolder(folderPath)
+                    On Error Resume Next
+                    Set folder = fso.GetFolder(folderPath)
+                    If Err.Number <> 0 Then
+                        Exit Sub
+                    End If
+                    
+                    ' Process files in current folder
+                    For Each file in folder.Files
+                        If LCase(fso.GetExtensionName(file.Name)) = "json" Then
+                            epoch = DateDiff("s", "01/01/1970 00:00:00", file.DateLastModified)
+                            outFile.WriteLine(epoch & " " & file.Path)
+                        End If
+                    Next
+                    
+                    ' Process subfolders recursively
+                    For Each subFolder in folder.SubFolders
+                        ProcessFolder(subFolder.Path)
+                    Next
+                End Sub
+                
+                ProcessFolder("%s")
+                outFile.Close
+            ]],
+			temp_out:gsub("\\", "\\\\"),
+			base_path:gsub("\\", "\\\\")
+		)
+
+		-- Create a second VBS script that will run the first one invisibly
+		local launcher_vbs = os.tmpname() .. "_launcher.vbs"
+		local launcher_script = string.format(
+			[[
+                Set WshShell = CreateObject("WScript.Shell")
+                WshShell.Run "wscript.exe //nologo %s", 0, True
+            ]],
+			temp_vbs
+		)
+
+		-- Write the scripts
+		suc, err = utils.write_file(temp_vbs, vbs_script)
+		if not suc then
+			wezterm.emit("resurrect.error", err)
+			return nil
+		end
+
+		suc, err = utils.write_file(launcher_vbs, launcher_script)
+		if not suc then
+			wezterm.emit("resurrect.error", err)
+			return nil
+		end
+		-- Execute using launcher (completely hidden)
+		os.execute("wscript.exe //nologo " .. launcher_vbs)
+
+		stdout, suc, err = utils.read_file(temp_out)
+
+		-- Clean up temp files
+		os.remove(temp_vbs)
+		os.remove(launcher_vbs)
+		os.remove(temp_out)
+
+		if suc then
+			return stdout
+		else
+			wezterm.emit("resurrect.error", err)
+			return nil
+		end
+	elseif utils.is_mac then
+		-- macOS recursive find command for JSON files
+		cmd = 'find "' .. base_path .. '" -type f -name "*.json" -print0 | xargs -0 stat -f "%m %N"'
+	else
+		-- Linux optimized recursive find command for JSON files
+		cmd = string.format(
+			'find "$(realpath %q)" -type f -name "*.json" -printf "%%T@ %%p\\n" | awk \'{split($1, a, "."); print a[1], $2}\'',
+			base_path
+		)
+	end
+
+	-- Execute the command and capture stdout for non-Windows
+	stdout, suc, err = utils.execute(cmd)
+
+	if suc then
+		return stdout
```

Comment:
This needs to match the execute returns
```suggestion
	stdout, err = utils.execute(cmd)

	if err then
		return stdout
```